packages/model/src/user.ts

```diff
@@ -178,6 +181,7 @@ export type UserCreateDataObject = {
   role: Role;
   social?: Omit<UserSocialLinks, 'orcid'>;
   teams?: Pick<UserTeam, 'id' | 'role' | 'inactiveSinceDate'>[];
+  tags?: Pick<ResearchTagDataObject, 'id' | 'name'>[];
```

Comment:
although it might seem like it's correct because of the `teams` definition above, I actually think this field should just be `tagIds` similarly to how we have defined the input for the relationships elsewhere where the is no extra data on the relationship (`teams` has data on the 'linking model': `role` and `inactiveSinceDate`, they both need to be defined along with the `id`). 

Let me ask you this, could a user use an existing tag `id` and give it any arbitrary `name`?